package.json

```diff
@@ -79,6 +82,7 @@
     "node-sass": "^4.9.3",
     "nyc": "^13.0.1",
     "prettier": "^1.14.2",
+    "prop-types-to-dts": "^0.3.2",
```

Comment:
I'm really nervous about introducing a relatively unknown package like this. Your package looks fine but I'm not a typescript expert, and I have a hard time vetting it beyond a superficial level. Given that the main body of the work is done in just one file, will you check that one (index.js) file into the typegen folder so we can keep an eye on it?